nilearn/plotting/displays.py

```diff
@@ -519,7 +523,7 @@ def init_with_figure(cls, img, threshold=None,
         # People forget to turn their axis off, or to set the zorder, and
```

Comment:
set the border,

Reply:
forget to turn their axes off (not axis)

Reply:
I think it actually could be related to `zorder` here. And the `mpl` is `axis('off')` in mpl so I think that's okay, too. (I also did not modify this code/comment so maybe such general cleanup would be better in a separate PR.)

Reply:
Ah as in z_order.
> I  also did not modify this code/comment so maybe such general cleanup would be better in a separate PR.
Cool.